frameworklauncher/src/main/java/com/microsoft/frameworklauncher/webserver/LauncherExceptionHandler.java

```diff
@@ -60,11 +59,10 @@ public Response toResponse(Exception e) {
     // Map response status
     String logPrefix = "Http request failed due to: ";
     final int statusCode;
-    if (e instanceof SecurityException ||
-        e instanceof AuthorizationException) {
+    if (e instanceof SecurityException) {
       LOGGER.logInfo(e, logPrefix + "Unauthorized");
       statusCode = HttpStatus.SC_UNAUTHORIZED;
-    } else if (e instanceof ForbiddenException) {
+    } else if (e instanceof AuthorizationException) {
       LOGGER.logInfo(e, logPrefix + "Forbidden");
```

Comment:
>Forbidden [](start = 37, length = 9)

Message not consistent

Reply:
This is by designed.
The http status SC_UNAUTHORIZED is not right, it actually means authentication errors.

---
In reply to: [167493976](https://github.com/Microsoft/pai/pull/320#discussion_r167493976) [](ancestors = 167493976)